css/style.css

```diff
@@ -184,12 +227,119 @@ footer h3  {
 }
 
 
-
 footer  {
     
     left: 0;
     bottom: 0;
     width: 100%;
     text-align: center;
+}
+/* css for store.html */
+
+.rightcontent {
+    width: 50% ;
+    height:auto;
+}
+ 
+.secondpic {
+    width:100%;
+    height:auto;   
+}
+
+.content {
+    display: flex;
+    flex-direction: row-reverse;
+    
+}
+
+.title {
+    font-family: Roboto;
+    font-style: normal;
+    font-size: 30px;
+    color: #f36100f1;
+    font-weight: 900;
+/* border:2 px solid black; */
+
+}
+
+.leftcontent {
+    font-family: Roboto;
+    font-style: normal;
+    font-size: 15px;
+    color: #666666;
+    padding-left: 30px;
+    width: 50%;
+    height: auto;
+    padding-right: 40px;
+}
+
+fieldset {
+    border: 0;
+}
+
+.details {
+    display: flex;
+    justify-content: space-between;
+    
+}
+
+
+.address {
+    display: flex;
+    flex-direction: column;
+}
+
+
+
+.checkboxterms a {
+    color:#f36100f1;
+}
+
+ 
+.details input {
+    width: 235px;
+    height: 38px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
+/* for city form */
+
+.details select {
+    width: 235px;
+    height: 45px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
+
+.address input {
+    width: 515px;
+    height: 38px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
```

Comment:
Line 299 `.details input` and line 316 `.address input` are defining the `input` styles which are almost the same. 

We need to learn to reuse classes as much as possible. If we use one class per input and we have a very long form, then

    the style file gets huge and we don't want that because it affects user performance over time. These files needs to be sent over to the users' browsers.
    also, when a developer works in a company, in a team, we normally need to get back to these files to modify them. If you need to update the inputs in the form, we want to update one class and it updates all the inputs at once. But if we have multiple class per input, it will be pretty time consuming to update all the inputs one by one.

Solution:
Use one class for all the inputs in the code.
Use extra classes for any new css rules an input needs.

Code Pen: https://codepen.io/mahsa2-/pen/YzrPPom

Reply:
Inputs can have `padding` inside them so that when the user types, the text doesn't stick to the border and is visible to the user as they type.